pom.xml

```diff
@@ -7,7 +7,7 @@
 
     <groupId>com.ontotext</groupId>
     <artifactId>ontorefine-client</artifactId>
-    <version>1.0.0</version>
+    <version>1.1.0-SNAPSHOT</version>
```

Comment:
The convention is 1.1-SNAPSHOT, rather than 1.1.0-SNAPSHOT.

Reply:
Will fix it. Also I plan to add automatic version "control", when we release new versions in order to avoid errors like this one.

Reply:
Changed.